### ISSUE TITLE
Side-by-side default to toggle fullscreen

### ISSUE THREAD

**Issue by crynobone** [sparksuite/simplemde-markdown-editor#90]
Not sure if this is intended by it would be nice if it was configurable.

![screen shot 2015-09-07 at 10 29 39 am](https://cloud.githubusercontent.com/assets/172966/9708242/acc3a378-554b-11e5-9a13-c660f7e62513.png)
![screen shot 2015-09-07 at 10 29 48 am](https://cloud.githubusercontent.com/assets/172966/9708244/affaa03c-554b-11e5-971f-c333d78aeaf4.png)

**Comment by WesCossick** [comment]
So by design, side-by-side forces full screen because most embedded editors are not large enough to handle side-by-side well. Are your requesting an option to disable this behavior?

**Comment by crynobone** [comment]
> Are your requesting an option to disable this behavior?

Yes that would be nice. Or move it above everything else.

Alternatively it would be nice if we could listen to event when the editor enter fullscreen/exit fullscreen so we can tweak our display.

**Comment by WesCossick** [comment]
I'll consider these options for future development.

**Comment by dongwq** [comment]
+1, wish this feature.

**Comment by WesCossick** [comment]
After looking into this further, this is a possibility still, but I'm leaving it as an idea. It will require a decent rethinking of how the side by side mode behaves now, along with a rewrite of the CSS styling. If it's a really important feature to someone, please submit a PR with the necessary modifications.

**Comment by derryspann** [comment]
@WesCossick is it possible to simply auto update the preview pane that is typically hidden until the preview button is clicked?

Creating a side by side mode can be handled independently.
update: i am using this method instead. `SimpleMDE.prototype.markdown(markdowncontent)`

**Comment by WesCossick** [comment]
@hyperking Not sure if I understand what you're asking. Can you rephrase that?

**Comment by derryspann** [comment]
@WesCossick While typing markdown, I would like the html preview container to auto update without the need of clicking the  "eye" icon to preview the markdown in html.

does that help?

**Comment by WesCossick** [comment]
I'm not sure why you'd need to render the Markdown if the preview isn't visible.

**Comment by KrishnaPG** [comment]
+1 for this There should be a way to avoid the side-by-side option forcing the full screen always.

**Comment by volpeo** [comment]
+1 for this feature, one way or another

**Comment by gmolveau** [comment]
+1 also :)

**Comment by aruizca** [comment]
+1

**Comment by jstejada** [comment]
+1

**Comment by WesCossick** [comment]
I've locked this issue for further discussion. It has already been assigned to the milestone 2.0.0-beta1 and will be developed in the future.
